sdk/python/lib/pulumi/runtime/resource.py

```diff
@@ -434,7 +429,6 @@ def register_resource(res: 'Resource',
 
     async def do_register():
         try:
-            log.debug(f"preparing resource registration: ty={ty}, name={name}")
```

Comment:
Similarly redundant with the earlier `log.debug` call -- and Node.js doesn't have this additional log call.